chemprop/sklearn_train.py

```diff
@@ -136,6 +208,17 @@ def multi_task_sklearn(model: Union[RandomForestRegressor, RandomForestClassifie
     num_tasks = train_data.num_tasks()
 
     train_targets = train_data.targets()
+
+    if args.impute_mode:
+        train_targets = impute_sklearn(model=model,
+                                       train_data=train_data,
+                                       args=args,
+                                       logger=logger)
+    elif any(None in sublist for sublist in train_targets):
+        raise ValueError("Missing target values not tolerated for multi-task sklearn models." 
+                         "Use either --single_task to train multiple single-task models or impute"
+                         " targets via --impute_mode  <model/linear/median/mean/frequent>.")
```

Comment:
We could have it revert to single_task automatically if we wanted to. But now that we have impute_targets as a better option I don't think we'd ever want someone too. So this meaningful error with the options spelled out is great.